src/Avalonia.OpenGL/IGlContextExternalObjectsFeature.cs

```diff
@@ -23,23 +26,32 @@ public interface IGlContextExternalObjectsFeature
     public byte[]? DeviceUuid { get; }
 }
 
+//TODO12: Make private and expose IBitmapImpl-based import API for composition visuals
+[NotClientImplementable]
```

Comment:
Can you create an issue with brief API idea?

Reply:
The general idea is to have an interface similar to `ICompositionGpuInterop` that's can be consumed from the render thread by CustomCompositionVisualHandler (and other API that we may add in the future). Instead of returning an asynchronously imported object the import operation would be completed immediately and will provide in some way a render-thread bitmap object suitable for consumption with ImmediateDrawingContext